tracer/src/Datadog.Trace.Tools.Runner/Program.cs

```diff
@@ -148,6 +148,10 @@ private static void ConfigureApp(IConfigurator config, ApplicationContext applic
                   .WithDescription("Apply AOT automatic instrumentation on application folder")
                   .WithExample("apply-aot c:\\input\\ c:\\output\\".Split(' '))
                   .IsHidden();
+
+            config.AddCommand<AnalyzeInstrumentationErrorsCommand>("analyze-instrumentation")
+                  .WithDescription("Analyze instrumentation errors")
+                  .WithExample("analyze-instrumentation [--process-name dotnet] [--pid 12345] [--log-path \"C:\\ProgramData\\Datadog .NET Tracer\\logs\\\"]".Split(' '));
```

Comment:
Shouldn't this be 3 separate examples, for simplicity?

Reply:
For this I think I need to change it to a branch with three commands and three different examples,  which I'm not sure is fit here because is more like a one command with an optional arguments that why the arguments wrapped by '[' and ']'. WDYT?

Reply:
I don't think you need that - if you look at [this page](https://spectreconsole.net/cli/commands) it shows that it's fine to just chain several calls to `WithExample` one after the other - so I think we can just go for that?

Reply:
Fixed in f801e3c42919ad47e54a29ef80067ca56e79ac7a